src/passkey_child/passkey_child_common.c

```diff
@@ -229,6 +250,14 @@ check_arguments(const struct passkey_data *data)
         goto done;
     }
 
+    if (data->action == ACTION_FORMAT_PEM
+        && data->public_key == NULL) {
+        DEBUG(SSSDBG_OP_FAILURE,
+              "Too few arguments for format action.\n");
```

Comment:
Hi,

maybe you can explicitly say that `--public-key` is missing?

bye,
Sumit

Reply:
Fixed